src/components/svg-wrapper.tsx

```diff
@@ -249,6 +302,14 @@ const SvgWrapper = () => {
         }
     });
 
+    const calcSelectSX = () => roundToNearestN(selectStart.x <= selectMoving.x ? selectStart.x : selectMoving.x, 1);
```

Comment:
According to React's best practice, these calculations should be wrapped in `React.useEffect(() => {...}, [...selectStart, ...selectMoving])` as these numbers/functions should only be recalculated when `selectStart` and/or `selectMoving` are changed :)

Reply:
Should I write `const [SX, setSX] = React.useState(0);` and use `setSX()` in  useEffect()?

Reply:
Yes, this is more like some _React_ code. Or `const [selectCoord, setSelectCoord] = React.useState({sx:0, sy:0, ex:0, ey:0});` will save you some lines of code :)

Reply:
Hmm, I created 4 consts.

Reply:
No problems. I've got them covered.